projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/welcome/FirefoxDriverFactoryBean.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2013-2018 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package xxxxxx.yyyyyy.zzzzzz.selenium.welcome;
```

Comment:
- [ ] The `welcome` package is inappropriate.
Did you decide to put this class in?
This class is less descriptive and unkind, so you should write comments a little more.
It also affects the [guideline](http://terasolunaorg.github.io/guideline/5.4.1.RELEASE/ja/ImplementationAtEachLayer/CreateWebApplicationProject.html#selenium), so I think you can write it there.

Reply:
Changed the package that have placed.
After the merge, launch Issue in the guideline and add it.
5f5e6f6

Reply:
This class is deleted.
d6fd659